test/Dockerfile.ciphertest

```diff
@@ -0,0 +1,14 @@
+# FROM ubuntu:latest
```

Comment:
Didn't really want to create my own test image, but couldn't find the correct mix of tools otherwise.

Reply:
```
 docker build -f Dockerfile.ciphertest  -t quay.io/benjaminapetersen/console-ciphertest:latest .
```
and exists here https://quay.io/repository/benjaminapetersen/console-ciphertest

Reply:
I'm not sure we want to host this on your personal quay account. We could potentially use OpenShift builds to build the image.

Reply:
+1